fendermint/vm/core/src/event.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022-2024 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#[derive(strum_macros::Display)]
```

Comment:
I would move this to a top-level `fendermint/event` package, and rename it to `EventType`.